External/Plugins/ASCompletion/Model/MemberModel.cs

```diff
@@ -538,4 +538,40 @@ public int Compare(MemberModel a, MemberModel b)
         }
 
     }
+
+    /// <summary>
+    /// Compare members based on import name
+    /// </summary>
+    public class ByImportTypeMemberComparer : IComparer<MemberModel>
+    {
+        public static Int32 CompareImports(String import1, String import2)
+        {
+            // Use case-insensitive comparison.
+            //IComparer cmp = StringComparer.OrdinalIgnoreCase;
+            // Use case-sensitive comparison.
+            IComparer cmp = StringComparer.Ordinal;
```

Comment:
Good reminder: we generally don't properly support those, just kind of ignore this part of the import declaration. Normal insertions will work fine, but operations like Organize Imports will drop them.